firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/type/LiveClientSetupMessage.kt

```diff
@@ -19,19 +19,25 @@ package com.google.firebase.vertexai.type
 import kotlinx.serialization.ExperimentalSerializationApi
 import kotlinx.serialization.Serializable
 
+/**
+ * First message in a live session.
+ *
+ * Contains configuration that will be used for the duration of the session.
+ */
 @OptIn(ExperimentalSerializationApi::class)
 @PublicPreviewAPI
-internal class BidiGenerateContentClientMessage(
+internal class LiveClientSetupMessage(
```

Comment:
Why change the name? BidiGenerateContentClientMessage is the name in the proto

Reply:
See [b/410064582](https://b.corp.google.com/issues/410064582) for context.

There's a weird mix here where some stuff (internal or not) is labeled Bidi (which technically matches the proto), while other stuff is labeled Live (which matches the public api). Renaming them to Live makes it easier to find what's Live (ergo; bidi) related, as it's the same naming convention throughout the public api and internally.

It's also worth noting that the design doc labels these types under the "Live" prefix instead of "Bidi" as well.